cityofaustin/atd-moped
# 11428 edit component phase

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/11428

## Testing
**URL to test:** 


[11428_edit_component_phase--atd-moped-main.netlify.app/moped/](https://11428_edit_component_phase--atd-moped-main.netlify.app/moped/)

**Steps to test:**
1) create a new project component, toggle the 'use component phase' toggle and fill in a phase, subphase (phase needs to have associated subphases), and completion date. reopen and click 'edit attributes' to confirm.
2) edit an existing project subcomponent, change the phase (the subphase dropdown should repopulate) subphase, and completion date. save, reopen and click 'edit attributes' to confirm.
3) clear out data (clear specific fields, and/or untoggle the 'use component phase'). reopen and click 'edit attributes' to confirm.

---
#### Ship list
